migrations/1-3-2_1-3-3.lua

```diff
@@ -0,0 +1,10 @@
+for index, force in pairs(game.forces) do
+	force.reset_recipes()
+	force.reset_technologies()
+
+	local technologies = force.technologies
+	local recipes = force.recipes
+
+	recipes["circuit-1-b"].enabled = technologies["substrate_2"].researched
```

Comment:
Should also check, if the recipe and tech is not null, because it was not available before? That was at least another bug recently